client/src/modules/reducer/index.js

```diff
@@ -1,7 +1,9 @@
 import { combineReducers } from 'redux'
 
 import example from './example'
+import profile from './profile'
```

Comment:
I'm getting an error: Module not found: Can't resolve './profile'. I'm not sure if you meant to leave out the reducer from './profile'.

Reply:
Looks like I messed up and didn't add the reducer. I'll close this pull request and re-add it when its fixed.